prime-router/.vault/config/init.sh

```diff
@@ -28,6 +28,19 @@ else
   echo "CREDENTIAL_STORAGE_METHOD=HASHICORP_VAULT" >> /vault/env/.env.local
 fi
 
+# Create the TokenSigningSecret env var in the local env file, if its not there.
```

Comment:
Developers no longer have to create this key manually, it gets created during vault startup, if its not already there.